docs/operations_tasks/add_entry_to_aws_secrets.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Add an entry to our secrets stored on AWS
+parent: Operations tasks
+---
+
+# Add an entry to our secrets that is stored on AWS
+The below steps assume that you have the correct account privileges in AWS.
+
+1. Log in to your HCA's AWS account
+2. Make sure that you are at the right AWS region otherwise you won't able to see/manage our secrets stored in the Secret Manager.
+   The region should be set to: `us-east-1` (N. Virginia)
+3. Go to Secrets Manager. You can do it with any of the follong way:
+   - click `Secters Manager` on the recently visited links section if its is not your first time,
+   - select the Services drop-down and in the Security, Identity, & Compliance section click on `Secrets Manager` link,
+   - Type `Secters Manager` into the search bar and click on the appearing `Secrets Manager` link
+4. Click on the Secrets option on the left hand side menu bar
+5. Depending on which environment you would like to add the new secret you should type `ingest/{ENV_NAME}/secrets` to the search bar
+   where `ENV_NAME` is a placeholder and can be the following values: `dev`, `staging` and `prod`.
+6. Click on the name of the secret, for example `ingest/dev/secrets`.
+7. Scroll down to the `Secret value` section and click on the `Retrieve secret value` button.
+8. Click omn the `Edit` button and add your secrets as key/value pairs.
+9. When you finished adding all your new secrets click on the `Save` button to store it.
+
```

Comment:
could add reference to official AWS documentation https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_update-secret.html